source/CategoryTypes.h

```diff
@@ -0,0 +1,24 @@
+/* CategoryTypes.h
+Copyright (c) 2021 by Jonathan Steck
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef CATEGORY_TYPES_H_
+#define CATEGORY_TYPES_H_
+
+enum class CategoryType : int {
+	SHIP,
+	BAY_TYPE,
```

Comment:
Might as well just be `Bay` or `BAY`, since it will always be preceded by `CategoryType`